drlhp/pref_db.py

```diff
@@ -180,12 +185,16 @@ def get_dbs(self):
         return train_copy, val_copy
 
     def wait_until_len(self, min_len):
+        min_val_len = int(0.3*min_len)
         while True:
             self.lock.acquire()
             train_len = len(self.train_db)
             val_len = len(self.val_db)
             self.lock.release()
-            if train_len >= min_len and val_len != 0:
+            if train_len >= min_len: # and val_len > min_val_len:
```

Comment:
This is messier than how I want to leave the code, but I do think I'm okay with having this conditioned just on the size of the training set, rather than also requiring the sampling for the validation dataset to have reached some specific percentage of the training set